subsys/bluetooth/host/monitor.c

```diff
@@ -310,47 +310,6 @@ static inline uint8_t monitor_priority_get(uint8_t log_level)
 	return BT_LOG_DBG;
 }
 
-static void monitor_log_put(const struct log_backend *const backend,
```

Comment:
I think you meant to delete `monitor_log_process` instead.

Reply:
no, `put` is a legacy v1 backend API and process is the one working with v2.

Reply:
but at L360 (new version), you're referencing a function that you are deleting ?

Reply:
my bad. Fixed.